packages/server/src/instances/lifetime/service.ts

```diff
@@ -42,6 +42,7 @@ export class InstanceLifetimeService extends Service {
 
   async destroy() {
     for (const channel of this.channels.list()) {
+      // TODO: channel.scopes doesn't work
```

Comment:
What does that mean?

Reply:
That's a not I put there for another PR I noticed what we were doing here wasn't working

Reply:
I'll make the fix